### ISSUE TITLE
How to document properties with colon?

### ISSUE THREAD

**Issue by markcellus** [jsdoc/jsdoc#2094]
I recently have to work with RSS/XML objects that have colons in the property name e.g. `dc:creator`. How is this expressed in JSDocs? Whenever I try adding `dc:creator` to typed object, it's not recognized.

### Input code

```js
/**
 * @typedef {object} XMLChannel
 * @property {string} [dc:creator]
 */
```

### Expected behavior

Output and support by JSDoc integrated tools like VSCode 

### Current behavior

Outputs nothing + causes error in JSDoc supported tools like VSCode

**Comment by begin-again** [comment]
```js
/**
 * @typedef {{
 *  'dc:creator': string
 * }} XMLChannel
 */
```

**Comment by markcellus** [comment]
Yeah but I'd have to make a seperate typedef just for this property or change all of my other types to use this inline `{{}}` syntax just for this property.

I was hoping that there was a way to declare the property without changing the type syntax everywhere.
